src/main/scala/wordbots/CodeGenerator.scala

```diff
@@ -85,6 +88,8 @@ object CodeGenerator {
       case ChooseC(collection) => s"targets['choose'](${g(collection)})"
       case AllC(collection) => s"targets['all'](${g(collection)})"
       case RandomC(num, collection) => s"targets['random'](${g(num)}, ${g(collection)})"
+      case SpecificC() => s"targets['token']()"
+      case CopyOfC(objToCopy) => s"(x=>x.entries[0].card)(${g(objToCopy)})"
```

Comment:
Rather than explicitly writing JavaScript here (`x=>x.entries[0].card`) I'd prefer to keep this in wordbots-core and just do like `targets['copyOfCard']` or something here.

Even if the implementation is super simple now, it should still be future-proofed and it's easier to change and test this kind of stuff in `core` than in `parser`.